desktop/apps/page/test/end_to_end.js

```diff
@@ -0,0 +1,20 @@
+/* eslint-env mocha */
+import { setup, teardown } from '../../../../test/helpers/end_to_end'
+
+describe('Page app', () => {
+  let gravity, browser
+
+  before(async () => {
+    ({ gravity, browser } = await setup())
+    gravity.get('/api/v1/page/:id', (req, res) => {
+      res.send({ name: 'Terms', content: 'Hello *World*' })
+    })
+  })
+
+  after(teardown)
+
+  it('Render a markdown page', async () => {
+    const $ = await browser.page('/terms')
```

Comment:
I added this `page` method to the Nightmare instance for the convenience of getting a jQuery-like interface for a given path. Maybe this is more unnecessarily clever than it is helpful though?